AuxTel/Troubleshooting/ATCS/ATHexapod-fails-to-enable-with-the-rest-of-ATCS.rst

```diff
@@ -0,0 +1,93 @@
+.. This is a template for troubleshooting when some part of the observatory enters an abnormal state. This comment may be deleted when the template is copied to the destination.
+
+.. Review the README in this procedure's directory on instructions to contribute.
+.. Static objects, such as figures, should be stored in the _static directory. Review the _static/README in this procedure's directory on instructions to contribute.
+.. Do not remove the comments that describe each section. They are included to provide guidance to contributors.
+.. Do not remove other content provided in the templates, such as a section. Instead, comment out the content and include comments to explain the situation. For example:
+	- If a section within the template is not needed, comment out the section title and label reference. Include a comment explaining why this is not required.
+    - If a file cannot include a title (surrounded by ampersands (#)), comment out the title from the template and include a comment explaining why this is implemented (in addition to applying the ``title`` directive).
+
+.. Include one Primary Author and list of Contributors (comma separated) between the asterisks (*):
+.. |author| replace:: *Ioana Sotuela*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *Gonzalo Aravena*
+
+.. This is the label that can be used as for cross referencing this procedure.
+.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens.
+.. _ATHexapod-fails-to-enable-with-the-rest-of-ATCS:
+.. Each section should includes a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+#########################
+ATHexapod fails to enable with the rest of ATCS
+#########################
+
+
+.. _ATHexapod-fails-to-enable-with-the-rest-of-ATCS-Overview:
+
+Overview
+========
+
+When attempting to enable ATCS from the ATScriptQueue with the script `auxtel/enable_atcs.py`, ATHexapod fails to transition to enable.
```

Comment:
script names should be using the :file: format. 
`auxtel/enable_atcs.py` --- > :file:`auxtel/enable_atcs.py`

Reply:
ATScriptQueue ----> *LOVE/ATScriptQueue*